Source/Core/Common/SettingsHandler.cpp

```diff
@@ -11,9 +11,10 @@
 #include <string>
 
 #ifdef _WIN32
+#include <windows.h> //NOLINT
+// The following Windows headers must be included AFTER windows.h.
```

Comment:
OK. But I find it hard to believe you cannot write your own check to accomplish the same thing if the OS is Windows instead of *nix or OSX.